public/jsx/views/card-listing.jsx

```diff
@@ -23,25 +23,16 @@ module.exports = React.createClass({
   },
 
   componentDidMount: function() {
-    $.ajax({
-      method: 'get',
-      url: '/api/braintree/mozilla/paymethod/',
-      context: this,
-    }).then(function(data) {
-      if (this.isMounted()) {
-        // Ignore react/no-did-mount-set-state eslint warning
-        if (data.length) {
-          console.log('Card data found, showing card form');
-          this.setState({cards: data}); // eslint-disable-line
-        } else {
-          console.log('No card data found, showing card form');
-          this.transitionTo('card-form');
-        }
-      }
-    }).fail(function() {
-      // TODO: some error state.
-      console.log('Card list retrieval failed');
-    });
+    var user = UserStore.getLoggedInUser();
```

Comment:
might think about using a lighteight flux framework to be able to declaratively connect to stores and not have to depend on singletons. we're currently using flummox, but that has been recently deprecated for redux.